moped-editor/src/views/projects/projectsListView/ProjectsListViewFiltersConf.js

```diff
@@ -25,459 +23,493 @@ export const OPERATORS_WITHOUT_SEARCH_VALUES = [
-        "string_is_null",
-        "string_is_not_null",
-      ],
-    },
-    {
-      name: "updated_at",
-      label: "Modified",
-      placeholder: "Last modified",
-      type: "date",
-      operators: [
-        "date_equals",
-        "date_greater_than",
-        "date_greater_than_equal_to",
-        "date_less_than",
-        "date_less_than_equal_to",
-        "date_is_null",
-      ],
+  project_name_full: {
+    name: "project_name_full",
+    label: "Full name",
+    placeholder: "Enter full project name",
+    type: "string",
+    defaultOperator: "string_contains_case_insensitive",
+    operators: [
+      "string_contains_case_insensitive",
+      "string_equals_case_insensitive",
+      "string_does_not_equal_case_insensitive",
+      "string_begins_with_case_insensitive",
+      "string_ends_with_case_insensitive",
+    ],
+  },
+  project_name: {
+    name: "project_name",
+    label: "Name",
+    placeholder: "Enter project name",
+    type: "string",
+    defaultOperator: "string_contains_case_insensitive",
+    operators: [
+      "string_contains_case_insensitive",
+      "string_equals_case_insensitive",
+      "string_does_not_equal_case_insensitive",
+      "string_begins_with_case_insensitive",
+      "string_ends_with_case_insensitive",
+    ],
+  },
+  project_name_secondary: {
+    name: "project_name_secondary",
+    label: "Secondary name",
+    placeholder: "Enter secondary project name",
+    type: "string",
+    defaultOperator: "string_contains_case_insensitive",
+    operators: [
+      "string_contains_case_insensitive",
+      "string_equals_case_insensitive",
+      "string_does_not_equal_case_insensitive",
+      "string_begins_with_case_insensitive",
+      "string_ends_with_case_insensitive",
+    ],
+  },
+  project_description: {
+    name: "project_description",
+    label: "Description",
+    placeholder: "Enter project description",
+    type: "string",
+    defaultOperator: "string_contains_case_insensitive",
+    operators: [
+      "string_contains_case_insensitive",
+      "string_equals_case_insensitive",
+      "string_does_not_equal_case_insensitive",
+      "string_begins_with_case_insensitive",
+      "string_ends_with_case_insensitive",
+    ],
+  },
+  project_id: {
+    name: "project_id",
+    label: "ID",
+    placeholder: "Type Project ID number",
+    type: "number",
+    defaultOperator: "number_equals",
+    operators: [
+      "number_equals",
+      "number_does_not_equal",
+      "number_greater_than",
+      "number_greater_than_equal_to",
+      "number_less_than",
+      "number_less_than_equal_to",
+    ],
+  },
+  type_name: {
+    name: "type_name",
+    label: "Type",
+    placeholder: "Enter Project type",
+    type: "string",
+    defaultOperator: "string_contains_case_insensitive",
+    lookup: {
+      table_name: "moped_types",
+      getOptionLabel: (option) => option.type_name,
+      operators: ["string_contains_case_insensitive"],
```

Comment:
Forgive me for going back on the issue requirements but could you add `string_equals_case_insensitive` and `string_does_not_equal_case_insensitive` back in for `type_name` and `project_partners`? No change to default operator. Today's dev sync discussion made me realize this is actually taking away valid searches like "a project with one type that is _____".

Reply:
thanks for updating! 🙏